test/parallel/test-process-emit.js

```diff
@@ -18,3 +18,5 @@ process.on('SIGPIPE', common.mustCall((data) => {
 process.emit('normal', 'normalData');
 process.emit(sym, 'symbolData');
 process.emit('SIGPIPE', 'signalData');
+
+assert.strictEqual(Number.isNaN(process._eventsCount), false);
```

Comment:
Shouldn't you use global `isNaN()` here?  `Number.isNaN('boom')` would pass because it returns false.